src/containers/settings/EditSettingsScreen.js

```diff
@@ -683,6 +694,12 @@ class EditSettingsScreen extends Component {
           default: DEFAULT_APP_SETTINGS.serviceRibbonWidth,
           options: sidebarWidth,
         },
+        sidebarServicesLocation: {
+          label: intl.formatMessage(messages.sidebarServicesLocation),
+          value: settings.all.app.sidebarServicesLocation,
+          default: DEFAULT_APP_SETTINGS.sidebarServicesLocation,
+          options: sidebarServicesLocation,
```

Comment:
I think there's a way to not have the dropdown's default first entry (so in total, there will only be 3 entries as per your array, not 4). Please research and use that as well.

Reply:
This will be handled in a new commit